controllers/authorization.controller.js

```diff
@@ -0,0 +1,17 @@
+import { refreshToken, token } from "../utils/tokenGenerator.js";
+import {
+  accessTokenOptions,
+  refreshTokenOptions,
+} from "../constants/constants.js";
+
+export class AuthorizationController {
+  static async authCheck(req, res) {
```

Comment:
how do you check auth? I think You should get token from cookie decode them check expiration. If token was expired - refresh it using refresh token and update cookies. 
Not important thing: but you can implemented it :)
If are you interesting in you can use `passport` library.